hack/test/0000_00_clusters.open-cluster-management.io_managedclusters.crd.yaml

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: The RamenDR authors
```

Comment:
These are copied of CRD from other repositories, placed here for ease of use in envtests (mostly). I would hence assume these should not be copyrighted by Ramen.

Reply:
Having different copyrights is complicated, I don't think we want to go there.
It is easier to have different license for such files (this is why LICENSES/ is
a directory), but this also add complication for no good reason.

What is the issue of Ramen copyright and Apache license on all files in the repo?

Reply:
Both on DCO and these files, we copied them into this repository and do not own the copyright on them?

Reply:
the DCO is a special case since it is kind of a unmodifed license files and already 
has a copyright.

But file like this:
https://github.com/RamenDR/ramen/blob/aeb78e2c0b4865691806e8dd61d6247921c0d378/hack/test/0000_00_clusters.open-cluster-management.io_managedclusters.crd.yaml

has no info on the source of the file and the copyright. We cannot have such files in
the source without any copyright and license. What copyright and license do you want to use for this files?

Can we download the crds from the original source during the build like we do in #559
for rook yamls?